src/content/1.7/development/components/form/form-theme.md

```diff
@@ -0,0 +1,149 @@
+
+Example for the date widget::
+
+```twig
+{%- block date_widget -%}
+  {%- if widget == 'single_text' -%}
+    {{ block('form_widget_simple') }}
+  {%- else -%}
+    <div {{ block('widget_container_attributes') }}>
+      {{- date_pattern|replace({
+        '{{ year }}':  form_widget(form.year),
+        '{{ month }}': form_widget(form.month),
+        '{{ day }}':   form_widget(form.day),
+      })|raw -}}
+    </div>
+  {%- endif -%}
+{%- endblock date_widget -%}
+```
+
+## PrestaShop form themes
+
+PrestaShop Form themes are located in the `src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/` directory.
+
+### Original PrestaShop Form theme
+
+{{% notice warning %}}
+**This theme is limited and deprecated.**
+
+Originally, Symfony forms in PrestaShop were developed using the field-by-field rendering technique, where each field is rendered individually ([see example](https://github.com/PrestaShop/PrestaShop/blob/1.7.7.0/src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/employee_options.html.twig)). It's not possible to use the simplified `form_widget(form)` rendering method using this form theme, so it's progressively being replaced by a new one (read below).
+{{% /notice %}}
+
+This form theme is a copy of Symfony's original Bootstrap 4 form theme, with customizations made for PrestaShop.
+
+Its base theme is `bootstrap_4_layout.html.twig`. Used to render Symfony forms vertically, it relies on multiple files:
+
+- form_div_layout
+- typeahead
+- material
+
+By default, PrestaShop's Twig templates will use `bootstrap_4_horizontal_layout.html.twig` (as configured in PrestaShop's `/app/config/config.yml` file). This child of the above theme render forms horizontally instead of vertically.
+
+{{< figure src="../../img/old-form-theme-17.png" title="Original Form Theme example of rendering" >}}
+
+### PrestaShop UI Kit Form theme
+{{< minver v="1.7.7" title="true" >}}
+
+Starting on 1.7.7 [a new Form Theme][pr-begin-ui-kit-form-theme] has been built from scratch to support the simplified form rendering technique described in the example at the top of this article.
+
+Contrary to the original form theme, this theme _extends_ Symfony's Bootstrap 4 form theme, allowing it to inherit all improvements done to Symfony's own form theme since the original release of PrestaShop 1.7.0.
+
+`prestashop_ui_kit.html.twig` extends `prestashop_ui_kit_base.html.twig` and also relies on `bootstrap_4_horizontal_layout`, in order to render forms horizontally.
+
+Once all forms have been updated to work with the UI Kit Form Theme, it will become the default. Until then, Twig templates using this theme need activate it using this statement:
+
+```
+{% form_theme form 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit_base.html.twig' %}
+```
+
+{{< figure src="../../img/ui-kit-form-theme.png" title="UI Kit Form Theme example of rendering" >}}
+
+
+## Refactoring our forms to rely fully on the UI Kit Form Theme
```

Comment:
I believe this section is anecdotal and not documentation-worthy

Reply:
Removed in https://github.com/PrestaShop/docs/pull/1041/commits/3c9ace6ccb1fa225fd7c4f45c9ecac8bf3f2cafe